Chummer/frmOptions.cs

```diff
@@ -1867,11 +1900,13 @@ private void clbPlugins_ItemCheck(object sender, ItemCheckEventArgs e)
 
         }
 
-        private void chkUseLoggingApplicationInsights_CheckedChanged(object sender, EventArgs e)
+        private void cbUseLoggingApplicationInsights_SelectedIndexChanged(object sender, EventArgs e)
         {
             if (this._blnLoading)
                 return;
-            if (chkUseLoggingApplicationInsights.Checked)
+            EnumUseLoggingApplicationInsights useAI =
+                (EnumUseLoggingApplicationInsights)Enum.Parse(typeof(EnumUseLoggingApplicationInsights), ((ListItem)cbUseLoggingApplicationInsights.SelectedItem).Value.ToString());
```

Comment:
I think selectedItem should have the type `EnumUseLoggingApplicationInsights` already and you can just cast it?